### ISSUE TITLE
Icon Request: Patreon Duo Tone

### ISSUE THREAD

**Issue by MarcusTheSmith** [FortAwesome/Font-Awesome#16060]
**What can it be used for?**
Correct branding usage of Patreon logo
The latest Patreon branding requirements say that the brand logo can only be used in duotone. For people needing to display the Patreon logo, having a duo-tone version is a must if they want to use FA for displaying it. This would also push people in need of this icon to use Premium if they want to be brand compliant. 
https://www.patreon.com/brand

**Example image (optional)**

![image](https://user-images.githubusercontent.com/44681650/72260094-09b49f00-35e0-11ea-9293-b27b94d5783c.png)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

Brands icon are available in a different set from Duotone icons, and the latter are only available to Pro Users. If you need Patreon icon in two colors, please download the svg file from their website

Also, if you download the brand assets zip files provided at https://www.patreon.com/brand, you will find the monochrome logo in black color, so I think that the current Font Awesome implementation is fine

![image](https://user-images.githubusercontent.com/556268/72272091-5b811780-3628-11ea-879b-cacb7374ba41.png)

Hope it helps

Closing here

**Comment by MarcusTheSmith** [comment]
I don't think this should be closed because the given solution is not complete. Just because they have a PNG asset in black doesn't mean that their brand guidelines are moot. It specifically states "do not change the color of the Icon outside of the approved color palette" and shows the two options. There are plenty of brands that have or have-had duo-tone logos, I imagine that this isn't an unreasonable request. And I would totally get Pro if there was a Patreon Duo-tone, but there is not. Just like other icons have different options (some free and some Pro), why can't brands do the same for duo-tone? The PNG is a nice idea, but I am specifically using FontAwesome because I can dynamically render brand logos in a consistent way without having to store the assets. 

Does that make sense? Can we not close this, even if it isn't done right away? I really do think this could be valuable and looks better for Font Awesome for trying to provide options for people to stick as closely as possible to brand guidelines.

**Comment by tagliala** [comment]
> There are plenty of brands that have or have-had duo-tone logos, I imagine that this isn't an unreasonable request.

A request asking for a duotone version of the Patreon logo is more than reasonable! I would definitely have keep such a request open.

Please notice that the original post is about a violation of brand guidelines of the currently implemented Patreon icon.

> Just because they have a PNG asset in black doesn't mean that their brand guidelines are moot. It specifically states "do not change the color of the Icon outside of the approved color palette" and shows the two options.

It is possible that Patreon do not want to see their logo in black on a web page, but as long as they provide an official asset and they are using the white version in the footer, I would say that having a single-color version in the set is not clearly against brand guidelines.

![image](https://user-images.githubusercontent.com/556268/72286042-a0b34280-3644-11ea-8885-543b7ea95e5a.png)

> Can we not close this, even if it isn't done right away? I really do think this could be valuable and looks better for Font Awesome for trying to provide options for people to stick as closely as possible to brand guidelines.

I will reopen this as a request to implement all brands in duotone for Pro Users.

This new duo-tone brands set will have the following features:

- Will contain 438 new icons (at the moment I'm writing)
- Will be exclusive for Pro users
- Will require a lot of time for being implemented, so it will not be available soon

Feel free to edit the first post to request the whole set. Patreon logo is a good example of why that set of brand icons is needed

For the moment, if you must have the colored version of the Patreon logo, please use the official asset, it is the best we can do

**Comment by MarcusTheSmith** [comment]
> A request asking for a duotone version of the Patreon logo is more than reasonable! I would definitely have keep such a request open.
> 
> Please notice that the original post is about a violation of brand guidelines of the currently implemented Patreon icon.

Forgive the misunderstanding, I was not trying to say that the current one is directly against their guidelines, but that I want to be as compliant as possible with their desired branding, which a duo-toned logo seems to be best for. 

> It is possible that Patreon do not want to see their logo in black on a web page, but as long as they provide an official asset and they are using the white version in the footer, I would say that having a single-color version in the set is not clearly against brand guidelines.

This is a fair point and I would agree. I never noticed their footer before. It does make me a bit more comfortable using a single tone. Though, I still would like to implement with duo whenever they are finished. 

No rush, and thank you for continuing to improve the FA offerings. It sounds like this one is definitely an undertaking.

**Comment by tagliala** [comment]
> Forgive the misunderstanding, I was not trying to say that the current one is directly against their guidelines, but that I want to be **as compliant as possible with their desired branding**, which a duo-toned logo seems to be best for.

Thanks for the clarification, and forgive me too, I've misunderstood the first post.

I definitely agree with the emphasized part

**Comment by tagliala** [comment]
As an update,

https://www.patreon.com/brand

> DO NOT
> Use more than one color in the logo

(image upload does not work)

**Comment by MarcusTheSmith** [comment]
> As an update,
> 
> https://www.patreon.com/brand
> 
> > DO NOT
> > Use more than one color in the logo
> 
> (image upload does not work)

Haha, I guess eventually all logo decisions are old logo decisions. The irony is that their favicon still has the dual tone. I'm sure there was some big discussion in their branding department about how they could better represent the ethos of their company with a slight adjustment... again. haha
Thanks for the update!
